### PR TITLE
fix for war-2128 : Project File Creation App in Katana

### PR DESCRIPTION
**Pull request by apathapa-warrior**

### CODE REVIEW

**Comment by bjrr** [comment]
Please fix the subject to follow the PR subject convention @santhosh-warrior

File: katana/wapps/projects/static/projects/base_templates/Untitled.xml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" ?>
```
Comment (santhosh-warrior) [inline katana/wapps/projects/static/projects/base_templates/Untitled.xml]: The file name should be Project_Template.xml  not Untitled.xml.

File: katana/wapps/projects/suite_utils/defaults.py [hunk 2]
```diff
@@ -0,0 +1,132 @@
+from utils.dict_utils import invert_dict
```
Comment (santhosh-warrior) [inline katana/wapps/projects/suite_utils/defaults.py]: The directory Naming convention.  katana/wapps/project/project_utils/
rename suite_utils to project_utils

File: katana/wapps/projects/suite_utils/verify_suite_file.py [hunk 3]
```diff
@@ -0,0 +1,117 @@
+from collections import OrderedDict
```
Comment (santhosh-warrior) [inline katana/wapps/projects/suite_utils/verify_suite_file.py]: File name need to changed according to app.  verify_project_file.py  and also function within file need to be named correctly.

File: katana/wapps/projects/templates/projects/display_suite.html [hunk 4]
```diff
@@ -0,0 +1,67 @@
+<body>
```
Comment (santhosh-warrior) [inline katana/wapps/projects/templates/projects/display_suite.html]: Is the file required ?

Comment (rpenjuri-warrior) [reply katana/wapps/projects/templates/projects/display_suite.html]: Yes, File is required. Suite files should display in UI, after adding under suites section.

File: katana/config.json [hunk 5]
```diff
@@ -1,10 +1,10 @@
 {
     "engineer": "",
-    "pythonsrcdir": "",
-    "xmldir": "",
-    "testsuitedir": "",
-    "projdir": "",
-    "idfdir": "",
-    "testdata": "",
+    "pythonsrcdir": "/home/rpenjuri/Documents/Project_app_testing1/warriorframework_py3/warrior",
```
Comment (santhosh-warrior) [inline katana/config.json]: Why did you add the hardcode path in config.json ?
